src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Bugzilla/Bugzilla44461.cs

```diff
@@ -17,26 +17,21 @@ public Bugzilla44461UITests(TestDevice device)
 
 		public override string Issue => "ScrollToPosition.Center works differently on Android and iOS";
 
-		// Bugzilla44461 (src\Compatibility\ControlGallery\src\Issues.Shared\Bugzilla44461.cs)
 		[Test]
-		[FailsOnIOSWhenRunningOnXamarinUITest("This test is failing, likely due to product issue")]
```

Comment:
Can you keep these attributes here for now? 

It's fine to leave these here and include the if/defs as well

It's useful having these attributes here still because it keeps track of things that were working in Xamarin.Forms but are now failing on MAUI.

For a next step we'll want to work through these attributes and make sure to see if we need to create bugs where folks might hit issues when migrating, or if we want to fix the test.

Reply:
Yes, kept the attributes and committed the changes, can you please review and share your concerns